src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -563,6 +565,20 @@ def launch_fluent(
             kwargs = _get_subprocess_kwargs_for_fluent(env)
             if cwd:
                 kwargs.update(cwd=cwd)
+            if topy:
+                if isinstance(topy, str):
+                    journal_name = topy.split(".")[0]
```

Comment:
Here we have only 1 journal name either `topy=a.jou` or `topy=a.scm`, because `rsplit` is also gives list similar to `split` after splitting it on `.`

![image](https://user-images.githubusercontent.com/106588300/207350466-206076cb-09b9-44be-98ae-4f09f71f465c.png)